schema_based_on_diagram.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE "patients"
+(
+  "id" SERIAL PRIMARY KEY,
+  "name" VARCHAR(255),
+  "date_of_birth" DATE
+);
+
+CREATE TABLE "medical_histories"
+(
+  "id" SERIAL PRIMARY KEY,
+  "admitted_at" TIMESTAMP,
+  "patient_id" INTEGER,
+  "status" VARCHAR(255),
+  FOREIGN KEY ("patient_id") REFERENCES "patients" ("id")
+);
+
+CREATE INDEX ON "medical_histories"
+("patient_id");
+
+CREATE TABLE "treatments"
+(
+  "id" SERIAL PRIMARY KEY,
+  "type" VARCHAR(255),
+  "name" VARCHAR(255)
+);
```

Comment:
- You have done well so far, but you are yet to create a `many-many` relationship between the `medical_histories` table and the `treatments` table. You can implement this by creating a join table, `medical_histories_treatments`. Please fix this 😄 
- Do not forget to add indexes on the foreign keys for the join table, as required by the project requirements 👍 
```SQL  
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

Reply:
Thanks @mckent05 for noting this. 😄